src/gl_context.rs

```diff
@@ -65,16 +74,20 @@ impl<Native> GLContext<Native>
                attributes: GLContextAttributes,
                color_attachment_type: ColorAttachmentType,
                shared_with: Option<&Native::Handle>)
-        -> Result<GLContext<Native>, &'static str> {
-        Self::new_shared_with_dispatcher(size, attributes, color_attachment_type, shared_with, None)
+        -> Result<Self, &'static str> {
+        Self::new_shared_with_dispatcher(size,
```

Comment:
I'm a bit confused about this. `create_shared_with_dispatcher` calls `new_shared_with_dispatcher`, and `new_shared_with_dispatcher` calls `create_shared_with_dispatcher`, both seem unconditional.